src/shared/services/commandInterpreterHelper.js

```diff
@@ -337,9 +341,25 @@ const availableCommands = [
   },
   {
     name: 'cypher',
-    match: cmd => /^cypher$/.test(cmd),
+    match: cmd =>
+      /^cypher$/.test(cmd) ||
+      new RegExp(`^${autoCommitTxCommand}`, 'i').test(cmd),
     exec: (action, cmdchar, put, store) => {
       const state = store.getState()
+
+      // Since we now also handle queries with the :auto prefix, we need to strip that
+      // and attach to the actions object
+      const query = action.cmd.replace(
+        getCmdChar(state) + autoCommitTxCommand,
+        ''
+      )
+      action.query = query.trim()
+
+      // We need to find out if this is an impllicit tx or not
+      // i.e. Did we strip something off above here?
+      const autoCommit = action.query.length < action.cmd.trim().length
```

Comment:
You could get this value from the string.replace above
```
let autoCommit = false
const query = action.cmd.replace(
  getCmdChar(state) + autoCommitTxCommand,
  () => autoCommit = true && ''
)
```
But I'm not sure if it's more or less readable

Reply:
Yeah, I do think it's more readable the way it is tbh. Interesting idea though, I've never thought of that.